homeassistant/components/ssdp/__init__.py

```diff
@@ -112,9 +113,29 @@ async def _process_entries(self, entries):
                     )
                 )
 
-        await asyncio.wait(tasks)
+        if tasks:
+            await asyncio.wait(tasks)
```

Comment:
Let's change this to `asyncio.gather` or else exceptions will not be logged.

Reply:
👍  done